### DISCUSSION TITLE
2023/06/02/%E7%8E%B0%E5%9C%A8%E5%BC%80%E5%A7%8B%E5%87%BA%E9%A2%98/

### DISCUSSION THREAD

**Discussion by giscus** [FloranceYeh/FloranceYeh.github.io#12] [Announcements]
# 2023/06/02/%E7%8E%B0%E5%9C%A8%E5%BC%80%E5%A7%8B%E5%87%BA%E9%A2%98/

It's Florance!

https://florance.eu.org/2023/06/02/%E7%8E%B0%E5%9C%A8%E5%BC%80%E5%A7%8B%E5%87%BA%E9%A2%98/

**Comment by TRUTHluming** [comment]
烧饼

**Comment by FloranceYeh** [comment]
你再骂
